### DISCUSSION TITLE
Duckdns da rete locale

### DISCUSSION THREAD

**Discussion by zolabus** [indomus/forum#1660] [HUB: Home Assistant]
Ciao a tutti non riesco più ad accedere all'indirizzo duckdns dalla rete locale (credo) da quando mi hanno sostituito il router di Fastweb con il nuovo Nexxt. Da indirizzo locale tutto ok, se mi connetto con la vpn aziendale tutto ok se mi connetto con i dati da l cellulare tutt ok ma da pc e da cellulare su rete domestica niente.
Penso che siano a questo punto le porte su router ma come su quello precedente ho aperto la 443 sulla 8123
Il problema è che credo che questo infici anche l'integrazione tra alexa e HA
Qualche idea a a riguardo?
Grazie

**Comment by snakuzzo** [comment]
Se dici che non riesci a raggiungere HA dalla tua LAN utilizzando il FQDN, potrebbe essere un problema legato proprio alla sostituzione del router e della regola di [NAT loopback](https://andreabeggi.net/2007/11/05/nat-regola-di-loopback/)
